paddle/fluid/operators/distributed/grpc_client.cc

```diff
@@ -272,6 +277,31 @@ bool GRPCClient::Wait() {
   return ok_;
 }
 
+bool GRPCClient::Wait(std::shared_ptr<framework::BlockingQueue<int>> ret_q,
```

Comment:
This is not what we intended to do actually. We need to wait a specific request, not to wait some number of requests and we don't know what the request the waited response refer to.

Reply:
Thanks! Done!